docs/docs/reference/05-oauth-providers/mercado-libre.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
This file is not necessary, as we generate API reference from the source code.

See https://github.com/nextauthjs/next-auth/pull/6609/files#r1096711931